css/properties/max-height.json

```diff
@@ -162,6 +207,70 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "fit-content": {
+            "__compat": {
+              "description": "<code>fit-content</code>",
+              "support": {
+                "chrome": [
+                  {
+                    "version_added": "46"
+                  },
+                  {
+                    "prefix": "-webkit-",
+                    "version_added": "22"
+                  }
+                ],
+                "chrome_android": {
+                  "version_added": "46"
```

Comment:
This landed with a -webkit- prefix in 25.

Reply:
@jpmedley could you be more specific about your suggestion here (and the one below for webview)? Should it look like the `chrome` entry right before this one (i.e., two support statements, one for prefixed version_added @ 25 and one unprefixed @ 46)? Or do you want one entry (webkit prefixed, version_added @ 25)?

Reply:
This should match chrome exactly. I was listing what's missing.

Reply:
I believe I have got them all, but this PR has got somewhat messy.